docs/migration_guide.rst

```diff
@@ -6,6 +6,7 @@ Migration guide
 **********************
 
 0.15.0 makes the first step to remove the cairo-lang package as StarkNet.py dependency!
+It also adds initial support for Starknet 0.11.0 and related changes.
```

Comment:
I would change the order of these two sentences. From the user's perspective, 0.11 is more important